tera-templates/releases/header.html

```diff
@@ -0,0 +1,77 @@
+{#
+    Builds the header for the release dashboard
+    * `title` A string
+    * `description` A string
+    * `tab` A string with one of the following values
+        * `recent`
+        * `stars`
+        * `recent-failures`
+        * `failures`
+        * `activity`
+        * `queue`
+        * `author`
+    * `author` A string, used for the authors page
+#}
+{% macro header(title, description, tab, author=false) %}
+    <div class="cratesfyi-package-container">
+        <div class="container">
+            <h1 id="crate-title">{{ title }}</h1>
```

Comment:
`header.hbs` was incorporated into `base.html`. Because of how handlebars works, that file acted as a literal copy+paste into the file. `header.hbs` contained all the meta, js and css that's now in `base.html` and then pasted in `navigation.hbs`, which was basically an if statement between "release mode" and "package mode" with those all being pulled into independent templates because they are indeed independent